crates/cli/src/report.jinja

```diff
@@ -0,0 +1,298 @@
+                </label>
+            </div>
+            <div class="control-group">
+                <label>
+                    <input type="checkbox" id="hide-noisy" onchange="updateFilters()">
+                    Hide noisy tests (CV > 5%)
+                </label>
+            </div>
+            <div class="control-group">
+                <span id="visible-benchmarks"></span>
+            </div>
+        </div>
+
+        <table style="padding-top:20px" id="results-table">
+            <thead>
+                <tr>
+                    <th scope="col">Benchmark</th>
+                    <th scope="col">CV%</th>
+                    <th scope="col">{{ stats.baseline_prefix }} (baseline)</th>
+                    {% for prefix in stats.benchmarks[0].stats_by_prefix %}
+                        {% if prefix != stats.baseline_prefix %}
+                        <th scope="col">{{ prefix }}</th>
+                        {% endif %}
+                    {% endfor %}
+                </tr>
+            </thead>
+            <tbody>
+                {% for benchmark in stats.benchmarks %}
+                {% set bstats = benchmark.baseline_stats %}
+                {% set has_significant = namespace(value=false) %}
+                {% set best_confidence = namespace(value=1.0) %}
+                {% for prefix, pstats in benchmark.stats_by_prefix|items %}
+                    {% if prefix != stats.baseline_prefix and pstats.is_significant %}
+                        {% set has_significant.value = true %}
+                        {% if pstats.significance_level < best_confidence.value %}
+                            {% set best_confidence.value = pstats.significance_level %}
+                        {% endif %}
+                    {% endif %}
+                {% endfor %}
+                <tr class="benchmark-row"
+                    data-has-significant="{{ has_significant.value|lower }}"
+                    data-baseline-cv="{{ bstats.cv }}"
+                    data-benchmark-name="{{ benchmark.name }}">
+                    <td><a href="#{{ benchmark.name }}">{{ benchmark.name }}</a></td>
+                    {# CV% column #}
+                    {% set bstats = benchmark.stats_by_prefix[stats.baseline_prefix] %}
+                    {% set cv_class = "inconsistent" if bstats.cv > 5 else "" %}
+                    <td class="{{ cv_class }}">{{ bstats.cv|floatfmt }}%</td>
+                    {# Baseline column without CV #}
+                    <td class="cycle-count">{{ bstats.p25|intfmt }}</td>
+                    {# Then show other engines #}
+                    {% for prefix, pstats in benchmark.stats_by_prefix|items %}
+                        {% if prefix != stats.baseline_prefix %}
+                        {% set class = "slower" if pstats.is_significant and pstats.speedup_ratio and pstats.speedup_ratio > 1 else "faster" if pstats.is_significant and pstats.speedup_ratio and pstats.speedup_ratio < 1 else "" %}
```

Comment:
This seems redundant with the other conditionals; drop this and just hard-code the class.